test_command_line_tool.d

```diff
@@ -12,13 +12,35 @@ int main(string[] args)
     try { return tryMain(args); }
     catch(SilentException) { return 1; }
 }
+
 int tryMain(string[] args)
 {
-    foreach (entry; dirEntries("test", "*.har", SpanMode.shallow))
+    // TODO: move to std.path
+    version(Windows)
+        string exeExtention=".exe";
+    else
+        string exeExtention;
+
+    auto rootDir=__FILE_FULL_PATH__.dirName;
```

Comment:
Make it `auto rootDir = __FILE_FULL_PATH__.dirName.relativePath;`

Reply:
in combination with `dub run har:test_command_line_tool`  being run from other directories it would show `../` in paths which is not as clear as using absolute paths

unless I distinguish whether cwd == rootDir (not sure it adds much value though)